flexmeasures/data/models/planning/tests/conftest.py

```diff
@@ -219,6 +219,56 @@ def process(db, building, setup_sources) -> dict[str, Sensor]:
     return _process
 
 
+@pytest.fixture(scope="module")
+def add_stock_delta(db, add_battery_assets, setup_sources) -> dict[str, Sensor]:
+    """
+    Set up the same constant delta (capacity_in_mw) in different resolutions.
+
+    In 15 min event resolution, the maximum energy that the battery can produce/consume in period
+    is 0.25 * capacity_in_mw
+    """
+
+    battery = add_battery_assets["Test battery"]
+    capacity = battery.get_attribute("capacity_in_mw")
+    sensors = {}
+    sensor_specs = [
+        ("delta fails", timedelta(minutes=15), capacity * 1.2),
```

Comment:
Imo this became a lot more intuitive after switching the field unit from MWh to MW. The explanation, in lines 227-228, for why scheduling this sensor will always fail, should be updated accordingly, and can be elaborated a bit, too. Any loss higher than the capacity means that no schedule can ever compensate the loss, right? So a steady loss of 1.001 times the power capacity should also fail.

Reply:
True, it's too brief. I've updated the comment.